React/tic-tac/src/app/components/Router/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Game from '@screens/Game';
```

Comment:
The actual folder where the `index.js` of Game is, is somewhere else. Did you check this?